stdlib/3.7/dataclasses.pyi

```diff
@@ -0,0 +1,67 @@
+from typing import overload, Any, Callable, Dict, Generic, Iterable, Mapping, Optional, Tuple, Type, TypeVar, Union
+
+
+_T = TypeVar('_T')
+
+_DictFactory = Callable[List[Tuple[str, Any]], _T]
+_TupleFactory = Callable[List[Any], _T]
+
+class _MISSING_TYPE: ...
+MISSING: _MISSING_TYPE
+
+class _InitVarMeta(type): ...
+
+def asdict(obj: Any, *, dict_factory: _DictFactory = ...) -> _T: ...
```

Comment:
Needs to be `_DictFactory[_T]`. Maybe just skip the type alias since it's only used once and write `Callable...` here directly.